templates/taxi/manufacturer_list.html

```diff
@@ -1,50 +1,53 @@
-{% extends "base.html" %}
-
-{% block content %}
-  <h1>
-    Manufacturer List
-    <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary link-to-page">
-      Create
-    </a>
-  </h1>
-
-  {% if manufacturer_list %}
-    <table class="table">
-      <tr>
-        <th>ID</th>
-        <th>Name</th>
-        <th>Country</th>
-        <th>Update</th>
-        <th>Delete</th>
-      </tr>
-
-      {% for manufacturer in manufacturer_list %}
-        <tr>
-          <td>
-              {{ manufacturer.id }}
-          </td>
-          <td>
-              {{ manufacturer.name }}
-          </td>
-          <td>
-              {{ manufacturer.country }}
-          </td>
-          <td>
-              <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
-                Update
-              </a>
-            </td>
-            <td>
-              <a style="color: red"
-                href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">
-                Delete
-              </a>
-            </td>
-        </tr>
-      {% endfor %}
-    </table>
-
-  {% else %}
-      <p>There are no manufacturers in the service.</p>
-  {% endif %}
-{% endblock %}
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>
+    Manufacturer List
+    <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary link-to-page">
+      Create
+    </a>
+  </h1>
+
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+      <form method="get", action="{% url 'taxi:search-manufacturers' %}">
+        <input type="text", name="q", placeholder="Search manufacturers (name)">

```

Comment:
There are commas after the `method` and `name` attributes in the form and input tags. These should be removed to ensure proper HTML syntax.